.github/workflows/cd.yml

```diff
@@ -0,0 +1,21 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+
+  continuous_deployment:
+
+    runs-on: ubuntu-latest
+    name: Deployment
+
+    steps:
```

Comment:
My editor formatted these blank lines and also line 6 indentation (based on editorconfig I guess). Could you please check this :pleading_face: